website/docs/cloud-docs/users-teams-organizations/users.mdx

```diff
@@ -135,3 +135,30 @@ Protect your tokens carefully because they contain the same permissions as your
 To revoke a token, click the **trash can** next to it. That token will no longer be able to authenticate as your user account.
 
 -> **Note**: When SAML SSO is enabled there is a session timeout for user API tokens, forcing users to periodically reauthenticate through the web UI in order to keep their tokens active. Refer to [API Token Expiration](/terraform/enterprise/saml/login#api-token-expiration) for details.
+
+
+### GitHub App OAuth Token
+
+Click **Tokens** in the sidebar to manage your GitHub App token. This token is required for tasks such as connecting workspaces to an available GitHub App installation.
```

Comment:
```suggestion
Click **Tokens** in the sidebar to manage your GitHub App token. This token lets you connect a workspaces to an available GitHub App installation.
```

If the token has other jobs should we list them here explicitly?

Reply:
I think below we list the actions that require a GitHub App token